bin/main

```diff
@@ -1,66 +1,69 @@
 #!/usr/bin/env ruby
+require './lib/player.rb'
+require './lib/game.rb'
 
-# select game option method
-puts 'Welcome to Tic-Tac-Toe game! Please select your option'
-puts '1 - Play'
-puts '2 - Exit'
-option = gets.chomp.to_i
-puts option
-# select option method ends here
-
-if option == 2
-  puts 'Thank you! See you soon!!'
-elsif option == 1
-  puts 'Welcome!! Your match will start soon..'
-  # will call game start method here
-else
-  puts 'Invalid option. Please enter the correct option.'
-  # will call the game option method
+game = Game.new
+one = game.game_array
+def info(args, args1)
+  name = ''
+  puts 'Welcome to Tic-Tac-Toe Game!!'
+  loop do
+    puts 'Player1 enter your name.'
+    name = gets.chomp.capitalize
+    break if name != '' && name != ' '
+  end
+  args.player1.name = name
+  puts "Hi, #{args.player1.name} your key for the game is #{args.player1.key}"
+  loop do
+    puts 'Player2 enter your name.'
+    name = gets.chomp.capitalize
+    break if name != '' && name != ' '
+  end
+  args.player2.name = name
```

Comment:
💡 [OPTIONAL] There are repeating code blocks which is against DRY rules. You can convert these repeating code blocks into individual methods. You can call them repeatedly anywhere you want in your code.